content/en/platform/corda/5.0/deploying-operating/deployment/deploying/manual-bootstrapping.md

```diff
@@ -174,24 +174,37 @@ You can then execute the `create` script to create the topics.
 
-By default, a Corda installation automatically creates and populates the database schema it requires.
-To create the schema manually, do the following:
-1. Set the following override in the deployment configuration to disable the automatic creation:
+By default, a Corda installation automatically creates and populates the database schema it requires. 
 
-   ```yaml
-   bootstrap:
-     db:
-       enabled: false
-   ```
+{{< note >}}
+If you are deploying Corda Enterprise, you must disable automatic bootstrapping and manually configure the database, as described in this section.
+{{< /note >}}
 
-   {{< note >}}
+To create the schema manually, set the following override in the deployment configuration to disable the automatic creation:
 
-   * If you are applying SQL to a schema using the `psql` command you can specify which schema to apply it to using the `--dbname` parameter like so: `--dbname "dbname=cordacluster options=--search_path=<SCHEMA-NAME>"`
+```yaml
+bootstrap:
+   db:
+    enabled: false
+```
 
-   * If you are targeting schemas, database and crypto-generated SQL should be applied to the CONFIG schema, and create-user-config generated SQL should be applied to the RBAC schema. If the schemas are not specified, then the tables will be created in the default schema and the next steps in this procedure will need updating to reflect this.
-   {{< /note >}}
+Create and populate the database schema, as follows:
+1. [Create the Database Tables](#create-the-database-tables)
+2. [Populate the RBAC Database Connection Configuration](#populate-the-rbac-database-connection-configuration)
+3. [Populate the Crypto Database Connection Configuration](#populate-the-crypto-database-connection-configuration)
+4. [Populate the Virtual Nodes Database Connection Configuration](#populate-the-virtual-nodes-database-connection-configuration)
+5. [Populate the REST Admin User](#populate-the-rest-admin-user)
+6. [Create the RBAC and Crypto Users](#create-the-rbac-and-crypto-users)
+7. [Populate the Crypto Configuration](#populate-the-crypto-configuration)
 
-2. Use the Corda CLI to generate DML files for creating the database tables to use for each of the `crypto`, `config`, and `rbac` components.
+{{< note >}}
+* If you are applying SQL to a schema using the `psql` command, you can specify which schema to apply it to using the `--dbname` parameter: `--dbname "dbname=cordacluster options=--search_path=<SCHEMA-NAME>"`.
+* If you are targeting schemas, database and crypto-generated SQL should be applied to the `CONFIG` schema, and `create-user-config` generated SQL should be applied to the `RBAC` schema. If you do not specify the schema, the installation process creates the tables in the default schema and you must update the next steps in this procedure to reflect this.
+{{< /note >}}
+
+### Create the Database Tables
+
+1. Use the Corda CLI to generate DML files for creating the database tables to use for each of the crypto, config, and rbac components.
```

Comment:
"crypto", "config" and "rbac" not capitalized because you're talking about components here?